app/controllers/msb/SendTheLargestAmountsOfMoneyController.scala

```diff
@@ -69,7 +60,7 @@ class SendTheLargestAmountsOfMoneyController @Inject()(val authConnector: AuthCo
             msb <-
             cacheConnector.fetch[MoneyServiceBusiness](MoneyServiceBusiness.key)
             _ <- cacheConnector.save[MoneyServiceBusiness](MoneyServiceBusiness.key,
-              msb.sendTheLargestAmountsOfMoney(data)
+              msb.sendTheLargestAmountsOfMoney(Some(data))
```

Comment:
We could move the clear down into the model file? I don't know if this is best- let me know your thoughts.

Reply:
I spoke with @elkdanger with regards to this change. The above follows patterns used in other controllers.

Reply:
Cool. Looks good then 👍